src/features/embalm/stepContent/components/ArchaeologistListItem.tsx

```diff
@@ -33,9 +35,18 @@ export function ArchaeologistListItem({
 }: ArchaeologistListItemProps) {
   const { testDialArchaeologist } = useAttemptDialArchaeologists(setIsDialing);
   const dispatch = useDispatch();
-
+  const networkConfig = useNetworkConfig();
   const rowTextColor = isSelected ? (archaeologist.exception ? '' : 'brand.950') : '';
 
+  const { data } = useEnsName({
+    address: archaeologist.profile.archAddress as `0x${string}`,
+    chainId: networkConfig.chainId,
+  });
+
+  const formattedArchAddress = () => {
+    return data ?? formatAddress(archaeologist.profile.archAddress);
+  };
```

Comment:
from my understanding, this wagmi hook is async, so if there is an ENS name, we will display the the address momentarily until the ENS name has been resolved. I think that will work -- in the future, maybe we want to show a loading icons until we know whether a name(s) can be resolved.